js/interface.js

```diff
@@ -2336,10 +2331,10 @@ $('body').on('click', '[data-rule-edit]', function(event) {
 
   if (rule.exclude) {
     columnsListMode = 'exclude';
-    $('#excludes')[0].checked = true;
+    $('#exclude')[0].checked = true;
```

Comment:
This would fail silently in case `$('#exclude')` returns no matching elements.

```suggestion
    $('#exclude').prop('checked', true);
```

Reply:
@tonytlwu added changes as per your suggestion by @NikitaJadhavXoriant

Reply:
The change was even better than my suggestion 👍🏻